backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -259,9 +263,34 @@ public Person updateMe(
         residentCongregateSetting,
         employedInHealthcare);
     upsertPreferredLanguage(toUpdate, preferredLanguage);
+    updatePhoneNumbers(toUpdate, phoneNumbers);
     return _repo.save(toUpdate);
   }
 
+  /**
+   * This method updates the PhoneNumbers provided by adding/deleting them from the
+   * PhoneNumberRepository. It updates the PrimaryPhone on the Person, but does <em>not</em> save
+   * that object, expecting it to be saved upstream.
+   */
+  private void updatePhoneNumbers(Person person, List<PhoneNumber> incoming) {
+    if (incoming == null) {
+      return;
+    }
+    incoming.forEach(phoneNumber -> phoneNumber.setPerson(person));
+
+    var existingNumbers = person.getPhoneNumbers();
```

Comment:
`var`?

Reply:
Not a fan of `var` in general or do you feel it's particularly inappropriate here?